features/draft/spec/html.yml.dist

```diff
@@ -5068,8 +5068,9 @@ compat_features:
   #   safari_ios: "15.2"
   - api.crossOriginIsolated
 
-  # baseline: low
+  # baseline: high
   # baseline_low_date: 2022-01-11
+  # baseline_high_date: 2024-07-11
```

Comment:
I sent https://github.com/mdn/browser-compat-data/pull/23836 to tag html.elements.meta.name.color-scheme instead of letting it sit in this draft.